Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/opensaml/CertificateManagerImpl.java

```diff
@@ -57,10 +57,10 @@
     private KeyStore trustStore = null;
 
     public static final String TRUST_STORE_TYPE_KEY = "javax.net.ssl.trustStoreType";
-    public static final String TRUST_STORE_PASSWORD_KEY = "javax.net.ssl.trustStorePassword";
+    public static final String TRUST_STORE_SECRET_KEY = "javax.net.ssl.trustStorePassword";
```

Comment:
I'm not sure how the git-sonatype bot works; is there a way to flag false positives?  Making changes for the sake of fooling a static code analysis tool is a surefire way to decrease code quality -- but if y'all stand behind this change as an improvement to the code (or readability thereof), it should definitely stay in.